packages/cli-tools/bin/streamr-misc.ts

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env node
+import { program } from 'commander'
+import pkg from '../package.json'
+
+program
+    .version(pkg.version)
+    .usage('<command> [<args>]')
+    .description('miscellaneous subcommands')
+    .command('fetch-token', 'fetch a session token for the current user')
```

Comment:
How about renaming this to `get-session-token`? Would be in line with the method name. Maybe "token" is not specific enough in a crypto tool.

Reply:
Good point, token is a quite overloaded term so better to use `session-token`. And `fetch-session-token` is already pretty long so `get-session-token` better.